packages/m/mhook/xmake.lua

```diff
@@ -0,0 +1,20 @@
+package("mhook")
+
+    set_homepage("https://github.com/martona/mhook")
+    set_description("A Windows API hooking library ")
+
+    set_urls("https://github.com/apriorit/mhook/archive/$(version).zip")
```

Comment:
Please add another git url, we can select the given tag/branch from git url.

Reply:
Do you mean adding git@github.com:apriorit/mhook.git ?